pandas/core/arrays/numeric.py

```diff
@@ -199,3 +201,48 @@ def reconstruct(x):
             return tuple(reconstruct(x) for x in result)
         else:
             return reconstruct(result)
+
+    def _apply(self, func: Callable, **kwargs) -> NumericArray:
```

Comment:
can you add type parameters to Callable.

Reply:
I can replace it with `Callable[[np.ndarray,...],np.ndarray]`. Is it precise enough for you?

Reply:
Callable[[np.ndarray],np.ndarray] ?

Reply:
I need to add the `...` for the `**kwargs` given to the `func` call.